src/UXClient/Components/LineChart/LineChart.ts

```diff
@@ -1308,6 +1415,13 @@ class LineChart extends TemporalXAxisComponent {
             this.chartMargins.top += -28;
         }
 
+        // Check if any swimlane labels present & modify left margin if so
```

Comment:
IMO we should also only be adding the left margin if the axis type supports it.  so if we are using a global overlapped or shared axis, we should not be adding a left margin